modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.server/src/main/resources/application.conf

```diff
@@ -0,0 +1,252 @@
+#
+# Copyright (C) 2009-2016 Typesafe Inc. <http://www.typesafe.com>
+#
```

Comment:
I don't think it  is too relevant, this is just a configuration file. Actually I hoped that it could be read directly from the play jars, Maybe I'll try that and try removing this conf